_i18n/ar/resources/user-guides/change-restore-height.md

```diff
@@ -0,0 +1,39 @@
+{% include disclaimer.html translated="no" translationOutdated="no" %}
+
+## How to change wallet restore height
```

Comment:
This title is not necessary and should be removed

Reply:
All user guides start with the article title. Why this guide should be different?

Reply:
The title should be added in https://github.com/monero-project/monero-site/pull/1768/files#diff-21793b642d7e11b7e13418b727e75d96c62b4cf8d0d47100cda9a80ddb3d6fc0. I see now that it's missing. See for example https://raw.githubusercontent.com/monero-project/monero-site/master/resources/user-guides/importing_blockchain.md